cmd/meroxa/root/apps/deploy.go

```diff
@@ -113,6 +115,20 @@ func (d *Deploy) Flags() []builder.Flag {
 	return builder.BuildFlags(&d.flags)
 }
 
+func (d *Deploy) getDeployGitSha() string {
+	if v := os.Getenv(deployGitSha); v != "" {
+		return v
+	}
+	return d.config.GetString(deployGitSha)
+}
+
+func (d *Deploy) skipGitValidation() bool {
+	if v := os.Getenv(deploySkipGitValidation); v != "" {
+		return v == "true"
```

Comment:
Just asking out of curiosity -- could we also directly return `true` here or would this be different from `v == "true"`?

Reply:
Oh I think I got it now, the `v` will always return a stringified version of the boolean `true` and this is just making sure that the environment variable is set correctly -- nevermind me then! :-)